### ISSUE TITLE
Improve the accessibility of the report navigation links (e.g. system report, investment report, etc.)

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#214]
In our meeting with a section 508 accessibility POC, we noticed that he was unable to view the report links on the left sidebar of the site because it has an on-hover action to reveal itself. Therefore, he was unable to access any of the report links on the left side of the site. This is an accessibility issue that must be addressed, ideally, without altering the on-hover functionality of the sidebar.

**Comment by hatfieldjm4** [comment]
The side navbar accessibility will be put on-hold. We are replicating that functionality in the top navbar instead to give screen readers the same navigation capability.

**Comment by hatfieldjm4** [comment]
We couldn't find a solution to easily make the side navbar accessible, so we put the same functionality in the top navbar to make the report links accessible.
